### PR TITLE
Support configuring Resources and Instrumens per step

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: modules/server_new/src/main/scala/observe/server/ODBSequencesLoader.scala [hunk 1]
```diff
@@ -15,10 +15,12 @@ import observe.model.Observation
 import observe.model.Observer
 import observe.model.SystemOverrides
 
+import scala.annotation.unused
+
 final class ODBSequencesLoader[F[_]: Async](
-  odbProxy:   OdbProxy[F],
-  translator: SeqTranslate[F],
-  execEngine: ExecEngineType[F]
+  @unused odbProxy:   OdbProxy[F],
```
Comment (rpiaggio) [inline modules/server_new/src/main/scala/observe/server/ODBSequencesLoader.scala:21]: Thank you

File: modules/web/server/src/main/scala/observe/web/server/http4s/vars.scala [hunk 2]
```diff
@@ -27,8 +28,9 @@ trait EnumeratedVar[A: Enumerated]:
   def unapply(str: String): Option[A] =
     Enumerated[A].fromTag(str)
 
-object InstrumentVar extends EnumeratedVar[Instrument]
-object ResourceVar   extends EnumeratedVar[Resource]
+object InstrumentVar           extends EnumeratedVar[Instrument]
+object ResourceVar             extends EnumeratedVar[Resource]
+object ResourceOrInstrumentVar extends EnumeratedVar[Resource | Instrument]
```
Comment (rpiaggio) [inline modules/web/server/src/main/scala/observe/web/server/http4s/vars.scala:33]: `SubsystemVar` ?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: modules/web/server/src/main/scala/observe/web/server/http4s/vars.scala (hunk 2 above)
Comment (cquiroz) [reply modules/web/server/src/main/scala/observe/web/server/http4s/vars.scala:33]: 👍

**Pull request merged**